parm/config/gefs/config.waveinit

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
```

Comment:
Why is this being added?  Are we running with waves in GEFS at this point?

Reply:
we just copy that files from gfs when we were trying to run with wave.